src/main/java/mate.academy/service/ProductService.java

```diff
@@ -1,8 +1,10 @@
 package mate.academy.service;
 
 import java.util.List;
+import mate.academy.lib.Component;
 import mate.academy.model.Product;
 
+@Component
```

Comment:
The `@Component` annotation is applied to an interface. Typically, `@Component` should be used on concrete implementation classes rather than interfaces. Consider moving this annotation to the implementing class.